RELEASING.md

```diff
@@ -1,20 +1,19 @@
-# Releasing a new version
+# Release guide
 
-## 1) Update the image tags in `master`
+## Branching/tagging scheme
 
-In the latest `master` branch:
+Just like deploy-sourcegraph, we use version branches and version tags. We _additionally_ have a second set which is customer-replication branches and version tags:
 
-1. Find and replace `:insiders` with `:3.0.0` on all files except this one. This is required for the next step.
-2. Run these commands:
+- Tag examples: `v3.8.2`, `v3.9.2`, `customer-replica-v3.8.2`, `customer-replica-v3.9.2-2`
+- Branch examples: `3.8`, `3.9`, `3.8-customer-replica`, `3.9-customer-replica`
 
-```
-go get -u github.com/slimsag/update-docker-tags
-update-docker-tags --constraint 'sourcegraph/prometheus=<10.0' --constraint 'sourcegraph/grafana=<6.0' .
-```
+The customer replica ones are important as we must maintain some diffs for replication purposes, such as using a different Postgres version, changes to the `prometheus_targets.yml` and more. We use tags to ensure each AMI we save correlates directly with an immutable Git tag.
 
-3. Confirm the diff shows the image tags being updated to the version you expect, and push directly to `master`.
+## Releasing a new version
 
-## 2) Create the release branch:
+### Create the release branch
+
+> ⚠️ If you are using the Sourcegraph release tooling, this step will be done for you in the PR it creates.
```

Comment:
What is the "Sourcegraph release tooling" and where do I find it? Can we link to it here, or a handbook page, etc.?

Reply:
Added links in 5a5046e1359f447d0d08e6fd691f79496e69b894 !